mapstruct-protobuf3/spi-impl/src/main/java/org/mapstruct/example/protobuf/ProtobufAccessorNamingStrategy.java

```diff
@@ -0,0 +1,34 @@
+package org.mapstruct.example.protobuf;
+
+import org.mapstruct.ap.spi.DefaultAccessorNamingStrategy;
+
+import javax.lang.model.element.Element;
+import javax.lang.model.element.ElementKind;
+import javax.lang.model.element.ExecutableElement;
+import javax.lang.model.element.TypeElement;
+import javax.lang.model.type.TypeMirror;
+
+/**
+ * @author Thomas Kratz
+ */
+public class ProtobufAccessorNamingStrategy extends DefaultAccessorNamingStrategy {
+
+    public static final String PROTOBUF_GENERATED_MESSAGE_V3 = "com.google.protobuf.GeneratedMessageV3";
```

Comment:
Again something I read on the protbuf website (I have no real experience). It says that `GeneratedMessage` is an implementation detail and that the builders are implementing [`Message.Builder`](https://developers.google.com/protocol-buffers/docs/reference/java/com/google/protobuf/Message.Builder) or [`MessageLite.Builder`](https://developers.google.com/protocol-buffers/docs/reference/java/com/google/protobuf/MessageLite.Builder) depending on something you pick in the `.proto`